Modules/Devices/test/Devices.Domain.Tests/Identities/CancelDeletionProcessTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Backbone.BuildingBlocks.Domain;
+using Backbone.Modules.Devices.Domain.Entities.Identities;
+using Backbone.Tooling;
+using FluentAssertions;
+using Xunit;
+
+namespace Backbone.Modules.Devices.Domain.Tests.Identities;
+public class CancelDeletionProcessTests
+{
+    [Fact]
+    public void Should_cancel_deletion_process_during_grace_period()
+    {
+        // Arrange
+        var currentDate = SystemTime.UtcNow.Date;
+        SystemTime.Set(currentDate);
+
+        var activeIdentity = TestDataGenerator.CreateIdentity();
+        var activeDevice = new Device(activeIdentity);
+        activeIdentity.Devices.Add(activeDevice);
+
+        activeIdentity.StartDeletionProcessAsOwner(activeDevice.Id);
```

Comment:
This method returns the started deletion process. Why don't you save this in a variable so that you can use it in the assertions?